.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+on: [push]
+
+name: CI
+
+jobs:
+  build_and_test:
+    name: Rust project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --release --all-features
```

Comment:
What about adding extra clippy step?

```yaml
name: Build | Lint
uses: actions-rs/cargo@v1
with:
 command: clippy
 args: --all-targets --all-features -- -D clippy::all
```
@thash @jbpratt

Reply:
We absolutely can! I wanted to wait for most of the warnings to be fixed, but it doesn't really matter. Added!